app/assets/stylesheets/_design-system-waiting-room.scss

```diff
@@ -0,0 +1,84 @@
+// Remove after `basscss-sass/base-forms` is no longer in use (LG-3876). BassCSS applies default
+// styling to label elements, that this exists to unset to avoid conflicts with design system.
+
+
+// basscss-base-forms
+//------------------------------------------------
+input[type=text],
+input[type=date],
+input[type=datetime],
+input[type=datetime-local],
+input[type=email],
+input[type=month],
+input[type=number],
+input[type=password],
+input[type=search],
+input[type=tel],
+input[type=time],
+input[type=url],
+input[type=week] {
+  height: 3rem;
+  padding: 0.5rem 1rem;
+  vertical-align: middle;
+  -webkit-appearance: none;
+}
+
+// basscss-base-tables
+//------------------------------------------------
+table {
+  border-collapse: separate;
+  border-spacing: 0;
+  max-width: 100%;
+  width: 100%;
+}
+
+th {
+  text-align: left;
+  font-weight: bold;
+}
+
+th,
+td {
+  padding: .25rem 1rem;
+  line-height: inherit;
+}
+
+th { vertical-align: bottom }
+td { vertical-align: top }
+
+//basscss-base-topography
+//------------------------------------------------
+body {
+  font-family: "Source Sans Pro Web","Helvetica Neue","Helvetica","Roboto","Arial",sans-serif;
+  line-height: 1.5;
+  font-size: 100%;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: "Merriweather Web","Georgia","Cambria","Times New Roman","Times",serif;
+  font-weight: bold;
+  line-height: 1.5;
+  margin-top: 1em;
+  margin-bottom: 0.5em;
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 1rem;
+}
+
+dl, ol, ul {
+  margin-top: 0;
+  margin-bottom: 1rem;
+}
+
+pre, code, samp {
+  font-family:  'Source Code Pro', Consolas, monospace;
+  font-size: inherit;
+}
+
+pre {
+  margin-top: 0;
+  margin-bottom: 1rem;
+  overflow-x: scroll;
+}
```

Comment:
I don't believe we have any `<pre>` elements anywhere, and regardless, we already set [desired stylings through the design system](https://github.com/18F/identity-style-guide/blob/426ea919f5d7722da8165a5a5d57f9a59c834c7e/src/scss/components/_code.scss#L8-L35), so it should be safe to remove.

```suggestion
```

Reply:
Yeah I had searched this before and did not think we needed it. I just left it just in case. But we can take it out since we already set desired styling through the design system.